custom/icds_reports/utils/aggregation_helpers/agg_awc.py

```diff
@@ -469,6 +495,11 @@ def _launched_col(col):
             ('stadiometer', 'COALESCE(sum(stadiometer), 0)'),
             ('num_anc_visits', 'COALESCE(sum(num_anc_visits), 0)'),
             ('num_children_immunized', 'COALESCE(sum(num_children_immunized), 0)'),
+            ('state_is_test', lambda col: col if aggregation_level > 1 else "0"),
+            ('district_is_test', lambda col: col if aggregation_level > 2 else "0"),
+            ('block_is_test', lambda col: col if aggregation_level > 3 else "0"),
+            ('supervisor_is_test', lambda col: col if aggregation_level > 4 else "0"),
+            ('awc_is_test', lambda col: col if aggregation_level > 5 else "0")
```

Comment:
`aggregation_level` here will never be > 4 so these last 2 will always be 0. Is that correct?

Reply:
:+1: